.storybook/addons.jsx

```diff
@@ -22,5 +24,22 @@ const theme = create({
 });
 
 addons.setConfig({
-  theme
+  theme,
+  sidebar: {
+    renderLabel: ({ name, parameters = {} }) => {
+      const statusRegex = /\[([^)]+)\]/gi;
```

Comment:
Using the square brackets is obviously just a suggestion, we can choose anything we want